packages/ubuntu_wizard/pubspec.yaml

```diff
@@ -46,6 +49,7 @@ dev_dependencies:
   mockito: 5.1.0
   ubuntu_test:
     path: ../ubuntu_test
+  url_launcher_platform_interface: ^2.0.5  
```

Comment:
some more superfluous whitespaces to clean up while moving the `plugin_platform_interface` here :)

![image](https://user-images.githubusercontent.com/140617/171597512-86d5de6d-83e2-4b4e-ad92-557fc6dba757.png)